### PR TITLE
chore: RFC markdown file for discussion

### PR DESCRIPTION
**Pull request by NickTomlin**
I started with a discussion but realized that a PR is probably better suited for line-by-line conversation

### CODE REVIEW

File: TODO.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+This serves as a braindump for features that we want to scope for V1 (or seed a V2, V3, etc)
+
+# Project
+
+## Runtime
+
+- Node.js version? Do we have a minimum?
```
Comment (boney9) [inline TODO.md:7]: +1 on compatibility with 14+.

File: TODO.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  - @Nick: I would be fine with 14.x+ to jettison some legacy stuff but do not know if that would wall us off
+
+## Publication
+
+- Is there an NPM namespace / package we want to publish under? `@orkes/conductor-js` or `@conductor/client`
```
Comment (boney9) [inline TODO.md:12]: I think we can use the same github handle? conductor-sdks/.. Assuming we can also replicate this for other languages such as Clojure, Python etc.

File: TODO.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- prettier?
+  - helps reduce editor churn and space/tab bikeshedding
+  - [maybe use husky to automate](https://www.npmjs.com/package/husky)
+
+## Tests
+
+- Unit tests
+  - Framework: jest? Mocha?
+- Integration tests
+  - It'd be _awesome_ if we could use an actual conductor cluster running in docker for integration style tests
```
Comment (boney9) [inline TODO.md:31]: Huge plus one to this. I think we can easily provision and maintain this - similar to play.orkes.io.

**Comment by NickTomlin** [comment]
closing for housekeeping since we've talked offline and begun this with #4

**Pull request closed without merging**